src/v/pandaproxy/schema_registry/test/consume_to_store.cc

```diff
@@ -47,6 +47,31 @@ constexpr pps::schema_version version1{1};
 constexpr pps::schema_id id0{0};
 constexpr pps::schema_id id1{1};
 
+inline model::record_batch
+make_delete_subject_batch(pps::subject sub, pps::schema_version version) {
+    storage::record_batch_builder rb{
+      model::record_batch_type::raft_data, model::offset{0}};
+
+    rb.add_raw_kv(
+      to_json_iobuf(pps::delete_subject_key{.seq{0}, .node{0}, .sub{sub}}),
```

Comment:
Do the sequence and node need fixing? Is a TODO missing? Node 0 is a valid node, which might be confusing?

If the fields were optional, we could keep read compatibility and not present them with invalid values here.

Reply:
At this level of the unit testing, the node field doesn't matter, and the sequence just has to match the apparent sequence in the batch (which is always zero, because these batches aren't coming off a real topic).  So these are valid-but-unimportant values.